### DISCUSSION TITLE
Support of TDS Metr

### DISCUSSION THREAD

**Discussion by 19leva89** [arendst/Tasmota#11214] [Ideas]
Good afternoon, tell me, is it possible to support TDS Metr on Analog output A0? A ready-made example for ESP8266 is in this article. https://www.circuitschools.com/tds-meter-with-tds-sensor-and-arduino-for-water-quality-monitoring/
![TDS](https://user-images.githubusercontent.com/71250243/109859380-921f5480-7c65-11eb-8765-9af40bdfc813.png)

**Comment by sfromis** [comment]
Assuming a single analog output, no other pins used, within the supported voltage range (1v directly on the pin, or 3.3v on many dev boards) what you can do right away is to just use the regular ADC Input config for the analog input pin, and apply a formula (using rules or backend) to convert values to relevant units.
